test/langtools/tools/javac/patterns/DeconstructionPatternErrors.out

```diff
@@ -1,5 +1,7 @@
 DeconstructionPatternErrors.java:15:28: compiler.err.underscore.as.identifier
 DeconstructionPatternErrors.java:15:29: compiler.err.expected: token.identifier
+DeconstructionPatternErrors.java:43:37: compiler.err.illegal.start.of.type
```

Comment:
should error be more specific here? E.g. diamond not supported with type test pattern?